From 4d639d16d53337d29c5bdaccbf375de6b56cc737 Mon Sep 17 00:00:00 2001
From: Kunhui-Li <kunhuix.li@intel.com>
Date: Fri, 9 Jul 2021 11:31:34 +0800
Subject: [PATCH 2/2] config_tools: update board xml files for
 MAX_MSIX_TABLE_NUM fix

The 6236 PR has modified the board.xml format for MAX_MSIX_TABLE_NUM fix.
To compromise this PR, updates all the source file board.xmls.

Tracked-On: #6235
Signed-off-by: Kunhui-Li <kunhuix.li@intel.com>
---
 .../config_tools/data/adl-s-crb/adl-s-crb.xml | 392 ++++++++++--------
 1 file changed, 209 insertions(+), 183 deletions(-)

diff --git a/misc/config_tools/data/adl-s-crb/adl-s-crb.xml b/misc/config_tools/data/adl-s-crb/adl-s-crb.xml
index bd1d95bd7..1af17fe68 100644
--- a/misc/config_tools/data/adl-s-crb/adl-s-crb.xml
+++ b/misc/config_tools/data/adl-s-crb/adl-s-crb.xml
@@ -12,14 +12,14 @@
 	Version: 2
 	</BASE_BOARD_INFO>
   <PCI_DEVICE>
-	00:00.0 Host bridge: Intel Corporation Device 4660 (rev 01)
+	00:00.0 Host bridge: Intel Corporation Device 4660 (rev 02)
 	Region 3: Memory at &lt;ignored&gt; (32-bit, non-prefetchable)
-	00:02.0 VGA compatible controller: Intel Corporation Device 4680 (rev 04)
+	00:02.0 VGA compatible controller: Intel Corporation Device 4680 (rev 0c)
 	Region 0: Memory at 6000000000 (64-bit, non-prefetchable) [size=16M]
 	Region 2: Memory at 4000000000 (64-bit, prefetchable) [size=256M]
 	Region 0: Memory at 0000004010000000 (64-bit, non-prefetchable)
 	Region 2: Memory at 0000004020000000 (64-bit, prefetchable)
-	00:08.0 System peripheral: Intel Corporation Device 464f (rev 01)
+	00:08.0 System peripheral: Intel Corporation Device 464f (rev 02)
 	Region 0: Memory at 600113b000 (64-bit, non-prefetchable) [disabled] [size=4K]
 	00:0a.0 Signal processing controller: Intel Corporation Device 467d (rev 01)
 	Region 0: Memory at 6001120000 (64-bit, non-prefetchable) [size=32K]
@@ -31,45 +31,45 @@
 	Region 0: Memory at 600112c000 (64-bit, non-prefetchable) [disabled] [size=16K]
 	Region 2: Memory at 600113a000 (64-bit, non-prefetchable) [disabled] [size=4K]
 	00:15.0 Serial bus controller [0c80]: Intel Corporation Device 7acc (rev 10)
-	Region 0: Memory at 4017000000 (64-bit, non-prefetchable) [virtual] [size=4K]
+	Region 0: Memory at 4017000000 (64-bit, non-prefetchable) [disabled] [size=4K]
 	00:15.1 Serial bus controller [0c80]: Intel Corporation Device 7acd (rev 10)
-	Region 0: Memory at 4017001000 (64-bit, non-prefetchable) [virtual] [size=4K]
+	Region 0: Memory at 4017001000 (64-bit, non-prefetchable) [disabled] [size=4K]
 	00:15.2 Serial bus controller [0c80]: Intel Corporation Device 7ace (rev 10)
-	Region 0: Memory at 4017002000 (64-bit, non-prefetchable) [virtual] [size=4K]
+	Region 0: Memory at 4017002000 (64-bit, non-prefetchable) [disabled] [size=4K]
 	00:15.3 Serial bus controller [0c80]: Intel Corporation Device 7acf (rev 10)
-	Region 0: Memory at 4017003000 (64-bit, non-prefetchable) [virtual] [size=4K]
+	Region 0: Memory at 4017003000 (64-bit, non-prefetchable) [disabled] [size=4K]
 	00:16.0 Communication controller: Intel Corporation Device 7ae8 (rev 10)
 	Region 0: Memory at 6001135000 (64-bit, non-prefetchable) [size=4K]
 	00:17.0 SATA controller: Intel Corporation Device 7ae2 (rev 10)
-	Region 0: Memory at 50520000 (32-bit, non-prefetchable) [size=8K]
-	Region 1: Memory at 50524000 (32-bit, non-prefetchable) [size=256]
-	Region 5: Memory at 50523000 (32-bit, non-prefetchable) [size=2K]
+	Region 0: Memory at 4f920000 (32-bit, non-prefetchable) [size=8K]
+	Region 1: Memory at 4f924000 (32-bit, non-prefetchable) [size=256]
+	Region 5: Memory at 4f923000 (32-bit, non-prefetchable) [size=2K]
 	00:19.0 Serial bus controller [0c80]: Intel Corporation Device 7afc (rev 10)
-	Region 0: Memory at 4017004000 (64-bit, non-prefetchable) [virtual] [size=4K]
+	Region 0: Memory at 4017004000 (64-bit, non-prefetchable) [disabled] [size=4K]
 	00:19.1 Serial bus controller [0c80]: Intel Corporation Device 7afd (rev 10)
-	Region 0: Memory at 4017005000 (64-bit, non-prefetchable) [virtual] [size=4K]
-	00:1d.0 PCI bridge: Intel Corporation Device 7ab4 (rev 10)
+	Region 0: Memory at 4017005000 (64-bit, non-prefetchable) [disabled] [size=4K]
+	00:1a.0 PCI bridge: Intel Corporation Device 7ac8 (rev 10)
 	00:1e.0 Communication controller: Intel Corporation Device 7aa8 (rev 10)
-	Region 0: Memory at 4017006000 (64-bit, non-prefetchable) [size=4K]
+	Region 0: Memory at 4017006000 (64-bit, non-prefetchable) [disabled] [size=4K]
 	00:1e.3 Serial bus controller [0c80]: Intel Corporation Device 7aab (rev 10)
-	Region 0: Memory at 4017007000 (64-bit, non-prefetchable) [virtual] [size=4K]
+	Region 0: Memory at 4017007000 (64-bit, non-prefetchable) [disabled] [size=4K]
 	00:1f.0 ISA bridge: Intel Corporation Device 7a82 (rev 10)
 	00:1f.3 Multimedia audio controller: Intel Corporation Device 7ad0 (rev 10)
 	Region 0: Memory at 6001128000 (64-bit, non-prefetchable) [size=16K]
 	Region 4: Memory at 6001000000 (64-bit, non-prefetchable) [size=1M]
 	00:1f.4 SMBus: Intel Corporation Device 7aa3 (rev 10)
-	Region 0: Memory at 6001130000 (64-bit, non-prefetchable) [size=256]
+	Region 0: Memory at 6001130000 (64-bit, non-prefetchable) [disabled] [size=256]
 	00:1f.5 Serial bus controller [0c80]: Intel Corporation Device 7aa4 (rev 10)
-	Region 0: Memory at 50522000 (32-bit, non-prefetchable) [size=4K]
+	Region 0: Memory at 4f922000 (32-bit, non-prefetchable) [size=4K]
 	00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection (17) I219-V (rev 10)
-	Region 0: Memory at 50500000 (32-bit, non-prefetchable) [size=128K]
+	Region 0: Memory at 4f900000 (32-bit, non-prefetchable) [size=128K]
 	01:00.0 Non-Volatile memory controller: Intel Corporation SSD Pro 7600p/760p/E 6100p Series (rev 03)
-	Region 0: Memory at 50400000 (64-bit, non-prefetchable) [size=16K]
+	Region 0: Memory at 4f800000 (64-bit, non-prefetchable) [size=16K]
 	</PCI_DEVICE>
   <PCI_VID_PID>
-	00:00.0 0600: 8086:4660 (rev 01)
-	00:02.0 0300: 8086:4680 (rev 04)
-	00:08.0 0880: 8086:464f (rev 01)
+	00:00.0 0600: 8086:4660 (rev 02)
+	00:02.0 0300: 8086:4680 (rev 0c)
+	00:08.0 0880: 8086:464f (rev 02)
 	00:0a.0 1180: 8086:467d (rev 01)
 	00:12.0 0700: 8086:7af8 (rev 10)
 	00:14.0 0c03: 8086:7ae0 (rev 10)
@@ -82,7 +82,7 @@
 	00:17.0 0106: 8086:7ae2 (rev 10)
 	00:19.0 0c80: 8086:7afc (rev 10)
 	00:19.1 0c80: 8086:7afd (rev 10)
-	00:1d.0 0604: 8086:7ab4 (rev 10)
+	00:1a.0 0604: 8086:7ac8 (rev 10)
 	00:1e.0 0780: 8086:7aa8 (rev 10)
 	00:1e.3 0c80: 8086:7aab (rev 10)
 	00:1f.0 0601: 8086:7a82 (rev 10)
@@ -93,8 +93,8 @@
 	01:00.0 0108: 8086:f1a6 (rev 03)
 	</PCI_VID_PID>
   <WAKE_VECTOR_INFO>
-	#define WAKE_VECTOR_32          0x44A4000CUL
-	#define WAKE_VECTOR_64          0x44A40018UL
+	#define WAKE_VECTOR_32          0x43A4000CUL
+	#define WAKE_VECTOR_64          0x43A40018UL
 	</WAKE_VECTOR_INFO>
   <RESET_REGISTER_INFO>
 	#define RESET_REGISTER_ADDRESS  0xCF9UL
@@ -165,10 +165,27 @@
 	"Genuine Intel(R) 0000"
 	</CPU_BRAND>
   <CX_INFO>
-	/* Cx data is not available */
+	{{SPACE_FFixedHW, 0x00U, 0x00U, 0x00U, 0x00UL}, 0x01U, 0x01U, 0x00U},	/* C1 */
+	{{SPACE_SYSTEM_IO, 0x08U, 0x00U, 0x00U, 0x1815UL}, 0x02U, 0x7FU, 0x00U},	/* C2 */
+	{{SPACE_SYSTEM_IO, 0x08U, 0x00U, 0x00U, 0x1819UL}, 0x03U, 0x418U, 0x00U},	/* C3 */
 	</CX_INFO>
   <PX_INFO>
-	/* Px data is not available */
+	{0xBB9UL, 0x00UL, 0x0AUL, 0x0AUL, 0x003000UL, 0x003000UL},	/* P0 */
+	{0xBB8UL, 0x00UL, 0x0AUL, 0x0AUL, 0x001E00UL, 0x001E00UL},	/* P1 */
+	{0xAF0UL, 0x00UL, 0x0AUL, 0x0AUL, 0x001C00UL, 0x001C00UL},	/* P2 */
+	{0xA8CUL, 0x00UL, 0x0AUL, 0x0AUL, 0x001B00UL, 0x001B00UL},	/* P3 */
+	{0x9C4UL, 0x00UL, 0x0AUL, 0x0AUL, 0x001900UL, 0x001900UL},	/* P4 */
+	{0x960UL, 0x00UL, 0x0AUL, 0x0AUL, 0x001800UL, 0x001800UL},	/* P5 */
+	{0x898UL, 0x00UL, 0x0AUL, 0x0AUL, 0x001600UL, 0x001600UL},	/* P6 */
+	{0x834UL, 0x00UL, 0x0AUL, 0x0AUL, 0x001500UL, 0x001500UL},	/* P7 */
+	{0x76CUL, 0x00UL, 0x0AUL, 0x0AUL, 0x001300UL, 0x001300UL},	/* P8 */
+	{0x6A4UL, 0x00UL, 0x0AUL, 0x0AUL, 0x001100UL, 0x001100UL},	/* P9 */
+	{0x640UL, 0x00UL, 0x0AUL, 0x0AUL, 0x001000UL, 0x001000UL},	/* P10 */
+	{0x578UL, 0x00UL, 0x0AUL, 0x0AUL, 0x000E00UL, 0x000E00UL},	/* P11 */
+	{0x514UL, 0x00UL, 0x0AUL, 0x0AUL, 0x000D00UL, 0x000D00UL},	/* P12 */
+	{0x44CUL, 0x00UL, 0x0AUL, 0x0AUL, 0x000B00UL, 0x000B00UL},	/* P13 */
+	{0x3E8UL, 0x00UL, 0x0AUL, 0x0AUL, 0x000A00UL, 0x000A00UL},	/* P14 */
+	{0x320UL, 0x00UL, 0x0AUL, 0x0AUL, 0x000800UL, 0x000800UL},	/* P15 */
 	</PX_INFO>
   <MMCFG_BASE_INFO>
 	/* PCI mmcfg base of MCFG */
@@ -178,6 +195,9 @@
 	TPM2
 	</TPM_INFO>
   <CLOS_INFO>
+	rdt resources supported: L2
+	rdt resource clos max: 8
+	rdt resource mask max: '0x3ff'
 	</CLOS_INFO>
   <IOMEM_INFO>
 	00000000-00000fff : Reserved
@@ -185,44 +205,37 @@
 	0009f000-000fffff : Reserved
 	  000a0000-000bffff : PCI Bus 0000:00
 	  000f0000-000fffff : System ROM
-	00100000-3dff3fff : System RAM
-	3dff4000-3e00efff : Reserved
-	3e00f000-3e0c6fff : System RAM
-	3e0c7000-3e0c7fff : Reserved
-	3e0c8000-41646fff : System RAM
-	41647000-449b7fff : Reserved
-	449b8000-44aeffff : ACPI Non-volatile Storage
-	44af0000-44bfefff : ACPI Tables
-	44bff000-44bfffff : System RAM
-	44c00000-491fffff : Reserved
-	49e00000-503fffff : Reserved
-	  4c800000-503fffff : Graphics Stolen Memory
-	50400000-bfffffff : PCI Bus 0000:00
-	  50400000-504fffff : PCI Bus 0000:01
-	    50400000-50403fff : 0000:01:00.0
-	      50400000-50403fff : nvme
-	  50500000-5051ffff : 0000:00:1f.6
-	    50500000-5051ffff : e1000e
-	  50520000-50521fff : 0000:00:17.0
-	    50520000-50521fff : ahci
-	  50522000-50522fff : 0000:00:1f.5
-	    50522000-50522fff : 0000:00:1f.5 0000:00:1f.5
-	  50523000-505237ff : 0000:00:17.0
-	    50523000-505237ff : ahci
-	  50524000-505240ff : 0000:00:17.0
-	    50524000-505240ff : ahci
+	00100000-3d7bcfff : System RAM
+	3d7bd000-3d7d7fff : Reserved
+	3d7d8000-3d7dbfff : System RAM
+	3d7dc000-3d7dcfff : Reserved
+	3d7dd000-40500fff : System RAM
+	40501000-439b7fff : Reserved
+	439b8000-43aeffff : ACPI Non-volatile Storage
+	43af0000-43bfefff : ACPI Tables
+	43bff000-43bfffff : System RAM
+	43c00000-47ffffff : Reserved
+	48c00000-4f7fffff : Reserved
+	4f800000-bfffffff : PCI Bus 0000:00
+	  4f800000-4f8fffff : PCI Bus 0000:01
+	    4f800000-4f803fff : 0000:01:00.0
+	      4f800000-4f803fff : nvme
+	  4f900000-4f91ffff : 0000:00:1f.6
+	    4f900000-4f91ffff : e1000e
+	  4f920000-4f921fff : 0000:00:17.0
+	    4f920000-4f921fff : ahci
+	  4f922000-4f922fff : 0000:00:1f.5
+	  4f923000-4f9237ff : 0000:00:17.0
+	    4f923000-4f9237ff : ahci
+	  4f924000-4f9240ff : 0000:00:17.0
+	    4f924000-4f9240ff : ahci
 	c0000000-cfffffff : PCI MMCONFIG 0000 [bus 00-ff]
 	  c0000000-cfffffff : Reserved
 	e0690000-e069ffff : INTC1056:00
-	  e0690000-e069ffff : INTC1056:00 INTC1056:00
 	e06a0000-e06affff : INTC1056:00
-	  e06a0000-e06affff : INTC1056:00 INTC1056:00
 	e06b0000-e06bffff : INTC1056:00
-	  e06b0000-e06bffff : INTC1056:00 INTC1056:00
 	e06d0000-e06dffff : INTC1056:00
-	  e06d0000-e06dffff : INTC1056:00 INTC1056:00
 	e06e0000-e06effff : INTC1056:00
-	  e06e0000-e06effff : INTC1056:00 INTC1056:00
 	fec00000-fec003ff : IOAPIC 0
 	fed00000-fed003ff : HPET 0
 	  fed00000-fed003ff : PNP0103:00
@@ -237,63 +250,24 @@
 	fee00000-feefffff : pnp 00:05
 	  fee00000-fee00fff : Local APIC
 	ff000000-ffffffff : Reserved
-	100000000-8afbfffff : System RAM
-	  87d400000-87e4036d6 : Kernel code
-	  87e600000-87eb9ffff : Kernel rodata
-	  87ec00000-87ee859bf : Kernel data
-	  87f92e000-87fffffff : Kernel bss
-	8afc00000-8afffffff : RAM buffer
+	100000000-8b07fffff : System RAM
+	  7f5a00000-7f6800db6 : Kernel code
+	  7f6a00000-7f6f53fff : Kernel rodata
+	  7f7000000-7f727a83f : Kernel data
+	  7f7534000-7f79fffff : Kernel bss
+	8b0800000-8b3ffffff : RAM buffer
 	4000000000-7fffffffff : PCI Bus 0000:00
 	  4000000000-400fffffff : 0000:00:02.0
+	    4000000000-40007e8fff : efifb
 	  4010000000-4016ffffff : 0000:00:02.0
 	  4017000000-4017000fff : 0000:00:15.0
-	    4017000000-40170001ff : lpss_dev
-	      4017000000-40170001ff : i2c_designware.0 lpss_dev
-	    4017000200-40170002ff : lpss_priv
-	    4017000800-4017000fff : idma64.0
-	      4017000800-4017000fff : idma64.0 idma64.0
 	  4017001000-4017001fff : 0000:00:15.1
-	    4017001000-40170011ff : lpss_dev
-	      4017001000-40170011ff : i2c_designware.1 lpss_dev
-	    4017001200-40170012ff : lpss_priv
-	    4017001800-4017001fff : idma64.1
-	      4017001800-4017001fff : idma64.1 idma64.1
 	  4017002000-4017002fff : 0000:00:15.2
-	    4017002000-40170021ff : lpss_dev
-	      4017002000-40170021ff : i2c_designware.2 lpss_dev
-	    4017002200-40170022ff : lpss_priv
-	    4017002800-4017002fff : idma64.2
-	      4017002800-4017002fff : idma64.2 idma64.2
 	  4017003000-4017003fff : 0000:00:15.3
-	    4017003000-40170031ff : lpss_dev
-	      4017003000-40170031ff : i2c_designware.3 lpss_dev
-	    4017003200-40170032ff : lpss_priv
-	    4017003800-4017003fff : idma64.3
-	      4017003800-4017003fff : idma64.3 idma64.3
 	  4017004000-4017004fff : 0000:00:19.0
-	    4017004000-40170041ff : lpss_dev
-	      4017004000-40170041ff : i2c_designware.4 lpss_dev
-	    4017004200-40170042ff : lpss_priv
-	    4017004800-4017004fff : idma64.4
-	      4017004800-4017004fff : idma64.4 idma64.4
 	  4017005000-4017005fff : 0000:00:19.1
-	    4017005000-40170051ff : lpss_dev
-	      4017005000-40170051ff : i2c_designware.5 lpss_dev
-	    4017005200-40170052ff : lpss_priv
-	    4017005800-4017005fff : idma64.5
-	      4017005800-4017005fff : idma64.5 idma64.5
 	  4017006000-4017006fff : 0000:00:1e.0
-	    4017006000-40170061ff : lpss_dev
-	      4017006000-401700601f : serial
-	    4017006200-40170062ff : lpss_priv
-	    4017006800-4017006fff : idma64.6
-	      4017006800-4017006fff : idma64.6 idma64.6
 	  4017007000-4017007fff : 0000:00:1e.3
-	    4017007000-40170071ff : lpss_dev
-	      4017007000-40170071ff : pxa2xx-spi.7 lpss_dev
-	    4017007200-40170072ff : lpss_priv
-	    4017007800-4017007fff : idma64.7
-	      4017007800-4017007fff : idma64.7 idma64.7
 	  4020000000-40ffffffff : 0000:00:02.0
 	  6000000000-6000ffffff : 0000:00:02.0
 	  6001000000-60010fffff : 0000:00:1f.3
@@ -302,10 +276,6 @@
 	    6001100000-600110ffff : xhci-hcd
 	  6001110000-600111ffff : 0000:00:12.0
 	  6001120000-6001127fff : 0000:00:0a.0
-	    60011264d8-60011264e7 : pmt_telemetry.1.auto
-	    60011264e8-60011264f7 : pmt_telemetry.1.auto
-	    60011264f8-6001126507 : pmt_telemetry.1.auto
-	    6001126508-6001126517 : pmt_telemetry.1.auto
 	  6001128000-600112bfff : 0000:00:1f.3
 	    6001128000-600112bfff : ICH HD audio
 	  600112c000-600112ffff : 0000:00:14.2
@@ -316,20 +286,18 @@
 	  600113b000-600113bfff : 0000:00:08.0
 	</IOMEM_INFO>
   <BLOCK_DEVICE_INFO>
+	/dev/sdb3: TYPE="ext4"
 	/dev/nvme0n1p3: TYPE="ext4"
-	/dev/sda2: TYPE="ext4"
 	/dev/sda3: TYPE="ext4"
-	/dev/sda4: TYPE="ext4"
 	</BLOCK_DEVICE_INFO>
   <TTYS_INFO>
 	seri:/dev/ttyS0 type:portio base:0x3F8 irq:4
-	seri:/dev/ttyS4 type:mmio base:0x4017006000 irq:16 bdf:"00:1e.0"
 	</TTYS_INFO>
   <AVAILABLE_IRQ_INFO>
-	3, 5, 6, 7, 10, 11, 12, 13, 15
+	3, 5, 6, 7, 10, 11, 12, 13, 14, 15
 	</AVAILABLE_IRQ_INFO>
   <TOTAL_MEM_INFO>
-	32632524 kB
+	32636412 kB
 	</TOTAL_MEM_INFO>
   <CPU_PROCESSOR_INFO>
 	0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15
@@ -408,6 +376,7 @@
       <capability id="erms"/>
       <capability id="invpcid"/>
       <capability id="deprecate_fpu"/>
+      <capability id="rdt_a"/>
       <capability id="rdseed"/>
       <capability id="adx"/>
       <capability id="smap"/>
@@ -425,7 +394,6 @@
       <capability id="tme_en"/>
       <capability id="rdpid"/>
       <capability id="kl"/>
-      <capability id="cldemote"/>
       <capability id="movdiri"/>
       <capability id="movdiri64b"/>
       <capability id="pks"/>
@@ -520,6 +488,7 @@
       <capability id="erms"/>
       <capability id="invpcid"/>
       <capability id="deprecate_fpu"/>
+      <capability id="rdt_a"/>
       <capability id="rdseed"/>
       <capability id="adx"/>
       <capability id="smap"/>
@@ -537,7 +506,6 @@
       <capability id="tme_en"/>
       <capability id="rdpid"/>
       <capability id="kl"/>
-      <capability id="cldemote"/>
       <capability id="movdiri"/>
       <capability id="movdiri64b"/>
       <capability id="pks"/>
@@ -570,7 +538,7 @@
           <x2apic_id>0x0</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Core</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -582,7 +550,7 @@
           <x2apic_id>0x8</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Core</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -594,7 +562,7 @@
           <x2apic_id>0x10</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Core</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -606,7 +574,7 @@
           <x2apic_id>0x18</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Core</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -618,7 +586,7 @@
           <x2apic_id>0x20</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Core</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -630,7 +598,7 @@
           <x2apic_id>0x28</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Core</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -642,7 +610,7 @@
           <x2apic_id>0x30</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Core</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -654,7 +622,7 @@
           <x2apic_id>0x38</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Core</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -666,7 +634,7 @@
           <x2apic_id>0x40</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Atom</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -678,7 +646,7 @@
           <x2apic_id>0x42</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Atom</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -690,7 +658,7 @@
           <x2apic_id>0x44</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Atom</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -702,7 +670,7 @@
           <x2apic_id>0x46</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Atom</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -714,7 +682,7 @@
           <x2apic_id>0x48</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Atom</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -726,7 +694,7 @@
           <x2apic_id>0x4a</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Atom</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -738,7 +706,7 @@
           <x2apic_id>0x4c</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Atom</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -750,7 +718,7 @@
           <x2apic_id>0x4e</x2apic_id>
           <family_id>0x6</family_id>
           <model_id>0x97</model_id>
-          <stepping_id>0x1</stepping_id>
+          <stepping_id>0x2</stepping_id>
           <core_type>Atom</core_type>
           <native_model_id>0x1</native_model_id>
         </thread>
@@ -1072,6 +1040,11 @@
       <processors>
         <processor>0x0</processor>
       </processors>
+      <capability id="CAT">
+        <capacity_mask_length>10</capacity_mask_length>
+        <clos_number>8</clos_number>
+      </capability>
+      <capability id="CDP"/>
     </cache>
     <cache level="2" id="0x1" type="3">
       <cache_size>1310720</cache_size>
@@ -1087,6 +1060,11 @@
       <processors>
         <processor>0x8</processor>
       </processors>
+      <capability id="CAT">
+        <capacity_mask_length>10</capacity_mask_length>
+        <clos_number>8</clos_number>
+      </capability>
+      <capability id="CDP"/>
     </cache>
     <cache level="2" id="0x2" type="3">
       <cache_size>1310720</cache_size>
@@ -1102,6 +1080,11 @@
       <processors>
         <processor>0x10</processor>
       </processors>
+      <capability id="CAT">
+        <capacity_mask_length>10</capacity_mask_length>
+        <clos_number>8</clos_number>
+      </capability>
+      <capability id="CDP"/>
     </cache>
     <cache level="2" id="0x3" type="3">
       <cache_size>1310720</cache_size>
@@ -1117,6 +1100,11 @@
       <processors>
         <processor>0x18</processor>
       </processors>
+      <capability id="CAT">
+        <capacity_mask_length>10</capacity_mask_length>
+        <clos_number>8</clos_number>
+      </capability>
+      <capability id="CDP"/>
     </cache>
     <cache level="2" id="0x4" type="3">
       <cache_size>1310720</cache_size>
@@ -1132,6 +1120,11 @@
       <processors>
         <processor>0x20</processor>
       </processors>
+      <capability id="CAT">
+        <capacity_mask_length>10</capacity_mask_length>
+        <clos_number>8</clos_number>
+      </capability>
+      <capability id="CDP"/>
     </cache>
     <cache level="2" id="0x5" type="3">
       <cache_size>1310720</cache_size>
@@ -1147,6 +1140,11 @@
       <processors>
         <processor>0x28</processor>
       </processors>
+      <capability id="CAT">
+        <capacity_mask_length>10</capacity_mask_length>
+        <clos_number>8</clos_number>
+      </capability>
+      <capability id="CDP"/>
     </cache>
     <cache level="2" id="0x6" type="3">
       <cache_size>1310720</cache_size>
@@ -1162,6 +1160,11 @@
       <processors>
         <processor>0x30</processor>
       </processors>
+      <capability id="CAT">
+        <capacity_mask_length>10</capacity_mask_length>
+        <clos_number>8</clos_number>
+      </capability>
+      <capability id="CDP"/>
     </cache>
     <cache level="2" id="0x7" type="3">
       <cache_size>1310720</cache_size>
@@ -1177,6 +1180,11 @@
       <processors>
         <processor>0x38</processor>
       </processors>
+      <capability id="CAT">
+        <capacity_mask_length>10</capacity_mask_length>
+        <clos_number>8</clos_number>
+      </capability>
+      <capability id="CDP"/>
     </cache>
     <cache level="2" id="0x8" type="3">
       <cache_size>2097152</cache_size>
@@ -1193,6 +1201,11 @@
         <processor>0x40</processor>
         <processor>0x42</processor>
       </processors>
+      <capability id="CAT">
+        <capacity_mask_length>16</capacity_mask_length>
+        <clos_number>16</clos_number>
+      </capability>
+      <capability id="CDP"/>
     </cache>
     <cache level="3" id="0x0" type="3">
       <cache_size>31457280</cache_size>
@@ -1221,21 +1234,21 @@
   </caches>
   <memory>
     <range start="0x0000000000000000" end="0x000000000009efff" size="651264"/>
-    <range start="0x0000000000100000" end="0x0000000041646fff" size="1096052736"/>
-    <range start="0x0000000044bff000" end="0x0000000044bfffff" size="4096"/>
-    <range start="0x0000000100000000" end="0x00000008afbfffff" size="33013366784"/>
+    <range start="0x0000000000100000" end="0x0000000040500fff" size="1077940224"/>
+    <range start="0x0000000043bff000" end="0x0000000043bfffff" size="4096"/>
+    <range start="0x0000000100000000" end="0x00000008b07fffff" size="33025949696"/>
   </memory>
   <devices>
-    <bus type="pci" address="0x0" id="0x4660">
+    <bus type="pci" address="0x0" id="0x4660" description="Host bridge: Intel Corporation">
       <vendor>0x8086</vendor>
       <identifier>0x4660</identifier>
       <subsystem_vendor>0x8086</subsystem_vendor>
       <subsystem_identifier>0x7270</subsystem_identifier>
       <class>0x060000</class>
       <resource type="memory" min="0xa0000" max="0xbffff" len="0x20000"/>
-      <resource type="memory" min="0x50400000" max="0xbfffffff" len="0x6fc00000"/>
+      <resource type="memory" min="0x4f800000" max="0xbfffffff" len="0x70800000"/>
       <resource type="memory" min="0x4000000000" max="0x7fffffffff" len="0x4000000000"/>
-      <device address="0x20000" id="0x4680">
+      <device address="0x20000" id="0x4680" description="VGA compatible controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x4680</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
@@ -1247,6 +1260,7 @@
         <capability id="Vendor-Specific"/>
         <capability id="PCI Express"/>
         <capability id="MSI">
+          <count>1</count>
           <capability id="per-vector masking"/>
         </capability>
         <capability id="Power Management"/>
@@ -1255,19 +1269,21 @@
         <capability id="PRI"/>
         <capability id="SR-IOV"/>
       </device>
-      <device address="0x80000" id="0x464f">
+      <device address="0x80000" id="0x464f" description="System peripheral: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x464f</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
         <subsystem_identifier>0x7270</subsystem_identifier>
         <class>0x088000</class>
         <resource type="memory" min="0x600113b000" max="0x600113bfff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
-        <capability id="MSI"/>
+        <capability id="MSI">
+          <count>1</count>
+        </capability>
         <capability id="Vendor-Specific"/>
         <capability id="Power Management"/>
         <capability id="Conventional PCI Advanced Features"/>
       </device>
-      <device address="0xa0000" id="0x467d">
+      <device address="0xa0000" id="0x467d" description="Signal processing controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x467d</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
@@ -1280,7 +1296,7 @@
         <capability id="Designated Vendor-Specific"/>
         <capability id="Designated Vendor-Specific"/>
       </device>
-      <device address="0x120000" id="0x7af8">
+      <device address="0x120000" id="0x7af8" description="Serial controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7af8</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
@@ -1290,7 +1306,7 @@
         <capability id="Power Management"/>
         <capability id="Vendor-Specific"/>
       </device>
-      <device address="0x140000" id="0x7ae0">
+      <device address="0x140000" id="0x7ae0" description="USB controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7ae0</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
@@ -1299,15 +1315,14 @@
         <resource type="memory" min="0x6001100000" max="0x600110ffff" len="0x10000" id="bar0" width="64" prefetchable="0"/>
         <capability id="Power Management"/>
         <capability id="MSI">
-          <capability id="multiple-message">
-            <count>8</count>
-          </capability>
+          <count>8</count>
+          <capability id="multiple-message"/>
           <capability id="64-bit address"/>
         </capability>
         <capability id="Vendor-Specific"/>
         <capability id="Vendor-Specific"/>
       </device>
-      <device address="0x140002" id="0x7aa7">
+      <device address="0x140002" id="0x7aa7" description="RAM memory: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7aa7</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
@@ -1317,47 +1332,47 @@
         <resource type="memory" min="0x600113a000" max="0x600113afff" len="0x1000" id="bar2" width="64" prefetchable="0"/>
         <capability id="Power Management"/>
       </device>
-      <device address="0x150000" id="0x7acc">
+      <device address="0x150000" id="0x7acc" description="Serial bus controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7acc</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
         <subsystem_identifier>0x7270</subsystem_identifier>
         <class>0x0c8000</class>
-        <resource type="memory" min="0x0" max="0xfff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
+        <resource type="memory" min="0x4017000000" max="0x4017000fff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
         <capability id="Power Management"/>
         <capability id="Vendor-Specific"/>
       </device>
-      <device address="0x150001" id="0x7acd">
+      <device address="0x150001" id="0x7acd" description="Serial bus controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7acd</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
         <subsystem_identifier>0x7270</subsystem_identifier>
         <class>0x0c8000</class>
-        <resource type="memory" min="0x0" max="0xfff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
+        <resource type="memory" min="0x4017001000" max="0x4017001fff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
         <capability id="Power Management"/>
         <capability id="Vendor-Specific"/>
       </device>
-      <device address="0x150002" id="0x7ace">
+      <device address="0x150002" id="0x7ace" description="Serial bus controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7ace</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
         <subsystem_identifier>0x7270</subsystem_identifier>
         <class>0x0c8000</class>
-        <resource type="memory" min="0x0" max="0xfff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
+        <resource type="memory" min="0x4017002000" max="0x4017002fff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
         <capability id="Power Management"/>
         <capability id="Vendor-Specific"/>
       </device>
-      <device address="0x150003" id="0x7acf">
+      <device address="0x150003" id="0x7acf" description="Serial bus controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7acf</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
         <subsystem_identifier>0x7270</subsystem_identifier>
         <class>0x0c8000</class>
-        <resource type="memory" min="0x0" max="0xfff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
+        <resource type="memory" min="0x4017003000" max="0x4017003fff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
         <capability id="Power Management"/>
         <capability id="Vendor-Specific"/>
       </device>
-      <device address="0x160000" id="0x7ae8">
+      <device address="0x160000" id="0x7ae8" description="Communication controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7ae8</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
@@ -1366,11 +1381,12 @@
         <resource type="memory" min="0x6001135000" max="0x6001135fff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
         <capability id="Power Management"/>
         <capability id="MSI">
+          <count>1</count>
           <capability id="64-bit address"/>
         </capability>
         <capability id="Vendor-Specific"/>
       </device>
-      <device address="0x170000" id="0x7ae2">
+      <device address="0x170000" id="0x7ae2" description="SATA controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7ae2</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
@@ -1379,40 +1395,43 @@
         <resource type="io_port" min="0x3060" max="0x307f" len="0x20" id="bar4"/>
         <resource type="io_port" min="0x3080" max="0x3087" len="0x8" id="bar2"/>
         <resource type="io_port" min="0x3088" max="0x308b" len="0x4" id="bar3"/>
-        <resource type="memory" min="0x50520000" max="0x50521fff" len="0x2000" id="bar0" width="32" prefetchable="0"/>
-        <resource type="memory" min="0x50523000" max="0x505237ff" len="0x800" id="bar5" width="32" prefetchable="0"/>
-        <resource type="memory" min="0x50524000" max="0x505240ff" len="0x100" id="bar1" width="32" prefetchable="0"/>
-        <capability id="MSI"/>
+        <resource type="memory" min="0x4f920000" max="0x4f921fff" len="0x2000" id="bar0" width="32" prefetchable="0"/>
+        <resource type="memory" min="0x4f923000" max="0x4f9237ff" len="0x800" id="bar5" width="32" prefetchable="0"/>
+        <resource type="memory" min="0x4f924000" max="0x4f9240ff" len="0x100" id="bar1" width="32" prefetchable="0"/>
+        <capability id="MSI">
+          <count>1</count>
+        </capability>
         <capability id="Power Management"/>
         <capability id="Reserved (0x12)"/>
       </device>
-      <device address="0x190000" id="0x7afc">
+      <device address="0x190000" id="0x7afc" description="Serial bus controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7afc</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
         <subsystem_identifier>0x7270</subsystem_identifier>
         <class>0x0c8000</class>
-        <resource type="memory" min="0x0" max="0xfff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
+        <resource type="memory" min="0x4017004000" max="0x4017004fff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
         <capability id="Power Management"/>
         <capability id="Vendor-Specific"/>
       </device>
-      <device address="0x190001" id="0x7afd">
+      <device address="0x190001" id="0x7afd" description="Serial bus controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7afd</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
         <subsystem_identifier>0x7270</subsystem_identifier>
         <class>0x0c8000</class>
-        <resource type="memory" min="0x0" max="0xfff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
+        <resource type="memory" min="0x4017005000" max="0x4017005fff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
         <capability id="Power Management"/>
         <capability id="Vendor-Specific"/>
       </device>
-      <device address="0x1d0000" id="0x7ab4">
+      <device address="0x1a0000" id="0x7ac8" description="PCI bridge: Intel Corporation">
         <vendor>0x8086</vendor>
-        <identifier>0x7ab4</identifier>
+        <identifier>0x7ac8</identifier>
         <class>0x060400</class>
-        <resource type="memory" min="0x50400000" max="0x504fffff" len="0x100000"/>
+        <resource type="memory" min="0x4f800000" max="0x4f8fffff" len="0x100000"/>
         <capability id="PCI Express"/>
         <capability id="MSI">
+          <count>1</count>
           <capability id="64-bit address"/>
         </capability>
         <capability id="Subsystem ID and Subsystem Vendor ID"/>
@@ -1427,23 +1446,28 @@
         <capability id="Physical Layer 16.0 GT/s"/>
         <capability id="Lane Margining at the Receiver"/>
         <bus type="pci" address="0x1">
-          <device address="0x0" id="0xf1a6">
+          <device address="0x0" id="0xf1a6" description="Non-Volatile memory controller: Intel Corporation SSD Pro 7600p/760p/E 6100p Series">
             <vendor>0x8086</vendor>
             <identifier>0xf1a6</identifier>
             <subsystem_vendor>0x8086</subsystem_vendor>
             <subsystem_identifier>0x390b</subsystem_identifier>
             <class>0x010802</class>
-            <resource type="memory" min="0x50400000" max="0x50403fff" len="0x4000" id="bar0" width="64" prefetchable="0"/>
+            <resource type="memory" min="0x4f800000" max="0x4f803fff" len="0x4000" id="bar0" width="64" prefetchable="0"/>
             <capability id="Power Management"/>
             <capability id="MSI">
-              <capability id="multiple-message">
-                <count>8</count>
-              </capability>
+              <count>8</count>
+              <capability id="multiple-message"/>
               <capability id="64-bit address"/>
               <capability id="per-vector masking"/>
             </capability>
             <capability id="PCI Express"/>
-            <capability id="MSI-X"/>
+            <capability id="MSI-X">
+              <table_size>16</table_size>
+              <table_bir>1</table_bir>
+              <table_offset>0x1000000</table_offset>
+              <pba_bir>1</pba_bir>
+              <pba_offset>0x0</pba_offset>
+            </capability>
             <capability id="Advanced Error Reporting"/>
             <capability id="Secondary PCI Express"/>
             <capability id="LTR"/>
@@ -1451,7 +1475,7 @@
           </device>
         </bus>
       </device>
-      <device address="0x1e0000" id="0x7aa8">
+      <device address="0x1e0000" id="0x7aa8" description="Communication controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7aa8</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
@@ -1461,24 +1485,24 @@
         <capability id="Power Management"/>
         <capability id="Vendor-Specific"/>
       </device>
-      <device address="0x1e0003" id="0x7aab">
+      <device address="0x1e0003" id="0x7aab" description="Serial bus controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7aab</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
         <subsystem_identifier>0x7270</subsystem_identifier>
         <class>0x0c8000</class>
-        <resource type="memory" min="0x0" max="0xfff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
+        <resource type="memory" min="0x4017007000" max="0x4017007fff" len="0x1000" id="bar0" width="64" prefetchable="0"/>
         <capability id="Power Management"/>
         <capability id="Vendor-Specific"/>
       </device>
-      <device address="0x1f0000" id="0x7a82">
+      <device address="0x1f0000" id="0x7a82" description="ISA bridge: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7a82</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
         <subsystem_identifier>0x7270</subsystem_identifier>
         <class>0x060100</class>
       </device>
-      <device address="0x1f0003" id="0x7ad0">
+      <device address="0x1f0003" id="0x7ad0" description="Multimedia audio controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7ad0</identifier>
         <subsystem_vendor>0x10ec</subsystem_vendor>
@@ -1489,10 +1513,11 @@
         <capability id="Power Management"/>
         <capability id="Vendor-Specific"/>
         <capability id="MSI">
+          <count>1</count>
           <capability id="64-bit address"/>
         </capability>
       </device>
-      <device address="0x1f0004" id="0x7aa3">
+      <device address="0x1f0004" id="0x7aa3" description="SMBus: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7aa3</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
@@ -1501,23 +1526,24 @@
         <resource type="io_port" min="0xefa0" max="0xefbf" len="0x20" id="bar4"/>
         <resource type="memory" min="0x6001130000" max="0x60011300ff" len="0x100" id="bar0" width="64" prefetchable="0"/>
       </device>
-      <device address="0x1f0005" id="0x7aa4">
+      <device address="0x1f0005" id="0x7aa4" description="Serial bus controller: Intel Corporation">
         <vendor>0x8086</vendor>
         <identifier>0x7aa4</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
         <subsystem_identifier>0x7270</subsystem_identifier>
         <class>0x0c8000</class>
-        <resource type="memory" min="0x50522000" max="0x50522fff" len="0x1000" id="bar0" width="32" prefetchable="0"/>
+        <resource type="memory" min="0x4f922000" max="0x4f922fff" len="0x1000" id="bar0" width="32" prefetchable="0"/>
       </device>
-      <device address="0x1f0006" id="0x1a1d">
+      <device address="0x1f0006" id="0x1a1d" description="Ethernet controller: Intel Corporation Ethernet Connection (17) I219-V">
         <vendor>0x8086</vendor>
         <identifier>0x1a1d</identifier>
         <subsystem_vendor>0x8086</subsystem_vendor>
         <subsystem_identifier>0x0000</subsystem_identifier>
         <class>0x020000</class>
-        <resource type="memory" min="0x50500000" max="0x5051ffff" len="0x20000" id="bar0" width="32" prefetchable="0"/>
+        <resource type="memory" min="0x4f900000" max="0x4f91ffff" len="0x20000" id="bar0" width="32" prefetchable="0"/>
         <capability id="Power Management"/>
         <capability id="MSI">
+          <count>1</count>
           <capability id="64-bit address"/>
         </capability>
       </device>
-- 
2.25.1

